velox/vector/SelectivityVector.cpp

```diff
@@ -27,4 +27,29 @@ SelectivityVector SelectivityVector::empty(vector_size_t size) {
   return SelectivityVector{size, false};
 }
 
+std::string SelectivityVector::toString(
+    vector_size_t maxSelectedRowsToPrint) const {
+  const auto selectedCnt = countSelected();
+
+  VELOX_CHECK_GE(maxSelectedRowsToPrint, 0);
+
+  std::stringstream out;
+  out << selectedCnt << " out of " << size() << " rows selected between "
+      << begin() << " and " << end();
+
+  if (selectedCnt > 0 && maxSelectedRowsToPrint > 0) {
+    out << ": ";
+    int cnt = 0;
```

Comment:
We should probably use something like this `bool firstValidEncountered = true;` and set `firstValidEncountered = false;` after the first value to make the intent explicit.

Reply:
Never mind. I missed that `cnt` is also used to limit the output. We don't need another variable for the delimiter.

Reply:
@majetideepak Thank you for reviewing, Deepak.